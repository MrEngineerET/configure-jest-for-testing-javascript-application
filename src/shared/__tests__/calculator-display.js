import React from 'react'
import {render} from '@testing-library/react'
import CalculatorDisplay from '../calculator-display'

test('renders calculator display', () => {
  const {container} = render(<CalculatorDisplay value="3" />)
  expect(container).toMatchSnapshot()
})
