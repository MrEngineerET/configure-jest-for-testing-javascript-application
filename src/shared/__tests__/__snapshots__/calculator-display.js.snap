// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders calculator display 1`] = `
<div>
  <div
    class="css-lq9ahq-calculator-display--CalculatorDisplay"
  >
    <div
      class="autoScalingText"
      data-testid="total"
      style="transform: scale(1,1);"
    >
      3
    </div>
  </div>
</div>
`;
